docs/pages/careers/developer-advocate.md

```diff
@@ -1,116 +1,111 @@
 
-- **Content**.
-  - Produce high-quality technical "how-to" content (blogs, webinars, demos, talks) addressing common user needs, latest technology advances, and emerging best practices. Videos might go to [our YouTube channel](https://www.youtube.com/@MUI_hq).
-  - Distribute content and leverage pieces to broaden awareness of the MUI brand, via existing connections.
-  - Rework the pages of the documentation that are confusing, base on feedback.
-- **Community**.
-  - Built our developer-focused community.
-  - Deliver presentations at meetups, conferences, and other ecosystem events.
-  - Create & produce events (podcasts, roundtables, hackathons).
-  - Manage the sponsorships of events, newsletters, etc.
-  - Help other team members grow at engaging with the community. For instance by encouraging and providing critical feedback on the blog posts they create, or by keeping track of the health of community contributions.
-- **Relationship-building**.
-  - Create relationships with fellow ecosystem influencers and open-source leaders.
-  - Nurture ongoing connections with user group members to build personal relationships and deeply understand their needs, usage, journeys, and barriers to adoption.
-- **Technical**.
-  - Write code where required to support how-to content, blog posts, and presentations.
-  - Create inspiring sample applications, documentation, and tutorials.
-  - Give feedback to product management, to influence the product roadmaps based on developers' needs.
-  - Follow GitHub issues to understand where developers face frustration and develop strategies to overcome these. This could be suggesting or implementing documentation updates, or proposing or contributing code changes that solve the core issue.
+### Key responsibilities
 
-For the right candidate:
+- Create educational materials (tutorials, how-to guides, demo apps) for devs of all skill levels
+- Seek out and implement improvements to the documentation
+- Contribute to the docs infrastructure and UX
+- Engage with the community and gather feedback for product teams
+- Serve as the "voice" of the company through social media, conferences, and events
 
-- Working with the Leadership to construct and execute on a hiring plan to grow the Developer Experience team.
+### Tasks and initiatives you might take on
 
-### Here are a few initiatives you might work on
+- Create tutorials for migrating from competitor products
+- Improve the Algolia search experience
+- Be a first responder for GitHub issues and PRs related to docs
+- Build example apps with MUI products and create how-to content around them
+- Research and write case studies
+- Overhaul the [Showcase](https://mui.com/material-ui/discover-more/showcase/)
+- Revamp the company blog infrastructure to empower less technical teammates to contribute
+- Solicit (and potentially contribute to) integrations with other popular libraries and frameworks
```

Comment:
Considering we want the person to be able to go as deep as fixing the product, I think we can keep it broad "contribute", in any way that make sense, mostly either doing if nobody else as the bandwidth or only connecting the dots if the person time is better spent somewhere else

```suggestion
- Contribute to integrations with other popular libraries and frameworks
```